core/chains/evm/txmgr/export_orm_test.go

```diff
@@ -0,0 +1,16 @@
+package txmgr
+
+// A way to export and test private methods and variables in the txmgr orm
+type Eorm struct {
+	*orm
+}
+
+func NewEorm(o ORM) *Eorm {
+	return &Eorm{o.(*orm)}
```

Comment:
type conversion to concrete type will allow us to test private methods on the orm struct. As code file becomes we should enforce practice of testing private functions

Reply:
Actually, is there a need to test private methods separately?
Ideally unit-tests should only test the public methods. This allows someone to do internal refactoring to make code more readable, without needing tests to be fixed everytime.

Reply:
I think when a private method is sufficiently complex, it becomes difficult to ensure functionality remains the same bc the public methods will rarely test the edge cases. I already found this to be the case when refactoring the code (that we missed things like returning nil vs erroring). Also, if a private method is sufficiently well-defined I think the code will undergo minimal refactoring. 

This is completely optional though and up to the person writing the code